### ISSUE TITLE
Run command before recover files

### ISSUE THREAD

**Issue by 1mm0rt41PC** [sandboxie-plus/Sandboxie#2202]
### Is your feature request related to a problem or use case?

I would like to use sandboxie for my family who is not security conscious. The use case is to verify that the recovered files are not malicious, I would like to be able to run SigmaRules or a SignCheck (Sysinternals) beforehand.

### Describe the solution you'd like

Add a section in the ini file:
```
RecoverChecker=C:\tools\sigcheck.exe -accepteula -vt -vrs %recover_file%
```
When a file is saved in a recoverable folder, run the `RecoverChecker` command:
- If the process exit with `0` => recover the file.
- If the process exit with another value => let the file inside te sandbox and show a popup to the user with the stdout of  the command from `RecoverChecker`. Maybe add a button to the window to force the recovery of the file.

### Describe alternatives you've considered

I tried to use inotify (https://pypi.org/project/inotify/) to detect the extracted files and run my checks but it's a bit dirty to do that on Windows. Antivirus can be a way, but it's quite easy to bypass one AV but hard to bypass multiple of them.

**Comment by DavidXanatos** [comment]
I was thinking about adding a built in ability to scan files using VT before recovering them, but then reports like these make me re thing if that's actually a good idea: https://github.com/winsiderss/systeminformer/discussions/1342
Now using a MSFT tool sounds like they probably will update it in time whenever required, so perhaps that would be an option worth considering, 
but then I'd say that 1/xx results does nor a virus make see this example https://www.virustotal.com/gui/file/700dea3e4012319a09ccadfce91cf090334cfe658d0bdc42204e77acbea1ef99/detection
So looking on the return value would be insufficient, one would need to parse the std out of that tool...

**Comment by 1mm0rt41PC** [comment]
I imagine a powershell script that do the trick:
```
param ($bin)
$sigcheck=sigcheck -vt -vrs -accepteula Sandboxie-Plus-x64-v1.3.2.exe $bin
if( -not $sigcheck.Contain(' 0/') ){
    Write-Output $sigcheck
    exit 1
}
$yara=yara C:\yara\rules\ $bin
if( -not $yara.Contain('malicious') ){
    Write-Output $yara
    exit 1
}
exit 0
```

and in the ini file:
```
[DefaultBox]
RecoverChecker=powershell -exec bypass -nop -File C:\tools\Malware-you-shall-not-pass.ps1 -bin %recover_file%
```

**Comment by DavidXanatos** [comment]
its available in 1.7.0

**Comment by bastik-1001** [comment]
How would a (similar) .BAT look like? Whenever it is even possible to use the CMD for it.
